unstructured/chunking/title.py

```diff
@@ -84,7 +84,7 @@ def chunk_by_title(
                                 item["start"] += start_char
                                 item["end"] += start_char
 
-                        if i > 0:
+                        if i > 0 and _value != value:
```

Comment:
what is this extra logic solving? adding specified data into other objects' metadata? asking since this doesn't seem to have anything to do with language so likely found a bug along the way?

Reply:
Yes, I can go add some more detailed comments as to why we need to add this but essentially if the list is `['eng']` and there's 15 or so elements, for each run of the loop, the number of elements in the list would double `['eng', 'eng']`. `['eng', 'eng', 'eng', 'eng']` etc. This would eventually cause a memory error if since the number of items in the list is 2 ^ n where n is the number of elements in the document. Perhaps this isn't the right fix, but this check prevents the list from extending itself with itself if the value is already there.

Reply:
then wouldn't it be better to not use `extend` below but use set operations to ensure we don't have duplicates? From the interface the value here for language can be a list of languages so we can have `['eng', 'ita']` as `_value` and `['eng']` as `value`. In this case they won't be equal so we'd extend one with the other which results in double `'eng'`.

Reply:
Yeah that's a great idea. I'll implement that. One of my concerns is that I'm not aware how else this function is used -- what other metadata list items are supposed to be run through this and is there expected behavior for those list items that I'm missing? I'm assuming there's proper test coverage for this but can never be certain. I think using set ops is a better alternative though and should maintain the desired functionality

Reply:
After further evaluation, because value could be any type, using a set will raise an exception because lists and dictionaries can't be hashed in a set operation. Instead we can iterate through the elements and append them to _value if they're not in the list.